templates/admin/event/edit.html.twig

```diff
@@ -0,0 +1,75 @@
+{% extends 'admin/base_admin.html.twig' %}
+
+{%block heading %}Редактировать событие{% endblock %}
+
+{% block content %}
+    {{ form_start(form) }}
+    {{ form_errors(form) }}
+    <div class="row py-3" >
+        <div class="col-6" >
+            {{ form_row(form.name) }}
+            {{ form_row(form.type) }}
+            {{ form_row(form.startDate) }}
+            {{ form_row(form.endDate) }}
+            {{ form_row(form.location) }}
+            {{ form_row(form.region) }}
+            <a href="#"> О жанрах и поджанрах</a>
```

Comment:
@daria-sieroshtan 
`<a href="#">` is generally bad, because `#` is not a valid link
MB you can replace it with proper `span`, or `h` or something?

Reply:
That's a placeholder for actual link, this static page is just not ready yet